integrationtest/src/test/java/org/hibernate/ogm/test/integration/redis/RedisModuleMemberRegistrationWithTTLConfiguredIT.java

```diff
@@ -11,6 +11,8 @@
 import javax.inject.Inject;
 import javax.persistence.EntityManagerFactory;
 
+import org.hibernate.SessionFactory;
+import org.hibernate.engine.spi.SessionFactoryImplementor;
```

Comment:
ah, now I get it, sorry! Looks like I didn't push my latest correctly, fixing now.. thanks a lot for noticing.

I should have noticed the build status :)